Day3/Kotlin/HammingDistance.kts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @author: d-l-mcbride
+ * @date: 10/12/2020
+ **/
+
+fun hammingDistance(first: String, second: String ):Int {
+    var distance = 0
```

Comment:
```suggestion
    if (first.length != second.length) {
       return -1
    }
    var distance = 0
```

Reply:
You need to check whether the strings are of equal length before you calculate the Hamming distance between them.

Reply:
@aaditkamat I have made the changes that were suggested

Reply:
@aaditkamat or @MadhavBahl I know you are probably busy but could you verify my changes, since I was trying to get my pull request done for Hacktoberfest. Unfortunately this pull request is actually many day2-4 (forgot to make branches for each). Thank you for your help!